usermods/Tubes/node.h

```diff
@@ -469,15 +411,60 @@ class LightNode {
         if (msg) {
             if(len == sizeof(NodeMessage)) {
                 instance->onPeerData(address, (const NodeMessage*)msg, len, rssi, true);
+            } else if (len == sizeof(wizmote_message)) {
```

Comment:
wizmote support is broke in the current main branch from the recent changes.  This one line change fixes that